### DISCUSSION TITLE
new setting

### DISCUSSION THREAD

**Discussion by EdgeDev01** [valinet/ExplorerPatcher#3538] [Ideas]
[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Control Center]
"UseLiteLayout"=dword:00000001 activates the windows 10 action center even on windows 11 24H2 canary builds! confirmed to work on windows 11 21H2 - 23H2 with no problems except a 3 second lag delay, so new potential option in settings

**Comment by EdgeDev01** [comment]
[potentialsettings.zip](https://github.com/user-attachments/files/16504526/potentialsettings.zip) here is the file for better examination

**Comment by pyrates999** [comment]
Since the code is not final and can change with each new release, we'd have to wait until this feature is in a beta version of widows 11.

**Reply by milestprower92** [reply]
> Since the code is not final and can change with each new release, we'd have to wait until this feature is in a beta version of widows 11.

ms brought back the 10 action center since 22H2 but this is not going to be on explorer patcher because microsoft can remove the 10 action center at any moment
